reddit_service_websockets/dispatcher.py

```diff
@@ -22,6 +46,14 @@ def __init__(self, metrics):
     def on_message_received(self, namespace, message):
         consumers = self.consumers.get(namespace, [])
 
+        # Compress the message
+        compressed = make_compressed_frame(message, COMPRESSOR)
```

Comment:
We can also test compression and if it comes out larger, just fall back to sending the raw version right?